sites/partners/src/components/listings/PaperListingForm/sections/ListingIntro.tsx

```diff
@@ -39,6 +44,31 @@ const ListingIntro = (props: ListingIntroProps) => {
         heading={t("listings.sections.introTitle")}
         subheading={t("listings.sections.introSubtitle")}
       >
+        <Grid.Row columns={1}>
```

Comment:
issue: This selection should only appear if the feature flag is turned on. The ticket has this mentioned in the "Technical Notes"  with the feature flag name of `showNonRegulatedFields`.  So for here it should show if any of the jurisdictions has the `showNonRegulatedFields` feature flag and then if a jurisdiction is selected only show this new field if that jurisdiction has the feature flag